flashinfer_trace/tests/references/test_gemm_n256_k2048.py

```
@@ -0,0 +1,75 @@
+def main():
+    print("Testing GEMM N=256, K=2048 Reference Implementation")
+
+    test_configs = [1, 4, 16, 64, 256]
+    passed = 0
+    total = len(test_configs)
+
+    for M in test_configs:
+        try:
+            if test_correctness(M):
+                passed += 1
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)